rundeckapp/src/main/groovy/org/rundeck/app/data/RdPageable.groovy

```diff
@@ -0,0 +1,16 @@
+package org.rundeck.app.data
+
+import org.rundeck.app.data.model.v1.page.Pageable
+
+class RdPageable implements Pageable{
```

Comment:
by default we should use `@CompileStatic` on all new groovy classes

Reply:
done